src/screens/Reader.tsx

```diff
@@ -93,6 +125,48 @@ export default function Reader() {
         );
     };
 
+    const openPrevChapter = useCallback(
```

Comment:
This and `openNextChapter` could be single function with offset parameter.

Reply:
done